pkg/test/main/main.go

```diff
@@ -133,13 +137,40 @@ func init() {
 
 	// Enable a muxed cache with partial snapshots
 	flag.BoolVar(&mux, "mux", false, "Enable muxed linear cache for EDS")
+
+	//
+	// These parameters control the the use of the pprof profiler
+	//
+
+	// Enable use of the pprof profiler
+	flag.BoolVar(&pprof_enabled, "pprof", false, "Enable use of the pprof profiler")
```

Comment:
I'd follow the previous naming patterns and do camelCase instead of snake case. 

```suggestion
flag.BoolVar(&pprofEnabled, "pprof", false, "Enable use of the pprof profiler")
```